statistics/handle/handle_test.go

```diff
@@ -2375,33 +2375,33 @@ func (s *testStatsSuite) TestIndexFMSketch(c *C) {
 		tk.MustExec("analyze table t")
 		rs := tk.MustQuery(fmt.Sprintf("select value from mysql.stats_fm_sketch")).Rows()
 		c.Assert(len(rs), Equals, rows)
+		totNDV := int64(0)
 		for i := range rs {
 			fm, err := statistics.DecodeFMSketch([]byte(rs[i][0].(string)))
 			c.Assert(err, IsNil)
+			totNDV += int64(ndv)
 			c.Assert(fm.NDV(), Equals, int64(ndv))
 		}
 	}
 
 	tk.MustExec("set @@tidb_enable_clustered_index=OFF")
-	tk.MustExec("create table t(a int, key(a))")
+	tk.MustExec("create table t(a int, key(a)) partition by hash(a) partitions 3")
 	tk.MustExec("insert into t values (1), (2), (2), (3)")
-	checkNDV(2, 3)
-	tk.MustExec("insert into t values (4), (5)")
-	checkNDV(2, 5)
+	checkNDV(6, 1)
```

Comment:
Should it be (6, 3) since there are 3 distinct values 1/2/3?

Reply:
From the `checkNDV()`, this should mean there are 6 fm sketch and 1 distinct value for each fm sketch.